### ISSUE TITLE
[BITV]: Note regarding FilePicker is not accessible

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#42592]
There are contrast issue and screenreader issues (no announce regarding a note)

![Screenshot from 2024-01-05 11-19-15](https://github.com/nextcloud/server/assets/6078378/9215b01b-7002-496f-99a2-8065c6297f4e)

Please create a same Note like in navigation bar settings (but probably should be an error and not a warning one):

![Screenshot from 2024-01-05 12-55-36](https://github.com/nextcloud/server/assets/6078378/31ebaf74-fb05-4888-8b0c-19ad48a87c8f)
